.travis.yml

```diff
@@ -1,22 +1,62 @@
 language: php
 
+stages:
+  - Smoke Testing
+  - Test
```

Comment:
Shouldn't this include "Test (with coverage)" as well?

Reply:
All stages that are not covered here runs after, implicitly. But I added all the stages explicitly, so done.